app/frontend/src/features/search/SearchResult.tsx

```diff
@@ -36,31 +51,61 @@ const useStyles = makeStyles((theme) => ({
 
 export default function SearchResult({ user }: { user: User.AsObject }) {
   const classes = useStyles();
+  const firstName = user.name.split(" ")[0];
+  const missingAbout = user.aboutMe.length === 0;
+  console.log(user);
   return (
     <Link to={routeToUser(user.username)} className={classes.root}>
       <Card className={classes.card}>
         <CardActionArea>
           <CardContent>
-            <div className={classes.resultHeader}>
-              <Avatar user={user} />
-              <Container>
-                <Typography variant="h2">{user.name}</Typography>
-                <Typography variant="subtitle1">
-                  {hostingStatusLabels[user.hostingStatus]}
+            <UserSummary user={user}>
+              {/* I think this container would sit more nicely inside UserSummary, to align 
+                  better with the Avater but I don't know whether we want to show
+                  hosting/meetup status in other areas we're using the UserSummary component */}
+              <Container
+                disableGutters={true}
+                className={classes.statusLabelWrapper}
+              >
+                <Typography
+                  className={classes.statusLabel}
+                  display="inline"
+                  variant="subtitle1"
+                  color="primary"
+                >
+                  {/* This "ask me about hosting/about meeting up" text can be changed when we add the icons in
+                      but at the moment just having Ask Me without a label is unclear" */}
+                  {hostingStatusLabels[user.hostingStatus] === "Ask me"
```

Comment:
```suggestion
                  {hostingStatusLabels[user.hostingStatus] === UNSURE
```
We're trying to not have loose strings around our code and trying to centralise them all in constants file so they can be found more easily/only need to change in one place if this message changes again etc.

Reply:
Yep good suggestion